src/main/webapp/resources/js/contexts/AnalysisDetailsContext.js

```diff
@@ -0,0 +1,131 @@
+import React, { useReducer, useContext } from "react";
```

Comment:
Code comments

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6